spec/menu_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'menu'
+
+describe Menu do
+  subject(:menu) { Menu.new }
```

Comment:
This is not needed as subject is always an instance of Menu when inside the describe block on line 3

Reply:
Amended now.